libraries/chain/include/eosio/chain/block_state_legacy.hpp

```diff
@@ -78,7 +75,7 @@ namespace eosio::chain {
          _cached_trxs = std::move( trxs_metas );
       }
 
-      bool                                                validated = false;
+      copyable_atomic<bool>                               validated{false};
```

Comment:
Would be good to mention this in the PR description.

Reply:
Indeed, done.